lib/react.dart

```diff
@@ -1766,6 +1791,18 @@ ComponentRegistrar2 registerComponent2 = (
   throw new Exception('setClientConfiguration must be called before registerComponent.');
 };
 
+/// Registers [componentFactory] on client.
+///
+/// Example:
+/// ```
+///   var myFunctionComponent = registerFunctionComponent((Map props) {
+///         return ['I am a function component', ...props.children];
+///       });
```

Comment:
#nit I could be wrong, but wouldn't the ending `});` usually has less indentation?
```suggestion
///   });
```

Reply:
If the triple backticks are being used, all of the indentation (lines 1798 and 1799 also) are off...